games/d.yaml

```diff
@@ -642,6 +642,21 @@
   updated: 2013-05-27
   url: https://dopewars.sourceforge.io/
 
+- name: doukutsu-rs
+  images:
+  - https://i.imgur.com/eZ0V5rK.png
+  lang:
+  - Rust
+  license:
+  - MIT
+  development: active
```

Comment:
```suggestion
  development: active
  status: unplayable
```

Reply:
is there a status for "playable but incomplete" or it counts as "unplayable"?

Reply:
https://github.com/opengaming/osgameclones/blob/906d30eca4e50596bd615053f867b57cfc2c191a/schema/games.yaml#L38

semi-playable